articles/quickstart/spa/react/01-login.md

```diff
@@ -12,6 +12,6 @@ github:
 contentType: tutorial
 useCase: quickstart
 ---
-<%= include('../_includes/_getting_started', { library: 'React', callback: 'http://localhost:3000/callback' }) %>
+<%= include('../_includes/_getting_started', { library: 'React', callback: 'http://localhost:3000/callback', showLogoutInfo: true, showWebOriginInfo: true }) %>
```

Comment:
@lbalmaceda assigning `true` to `showLogoutInfo` and `showWebOriginInfo` variables shows [logout URL](https://github.com/auth0/docs/blob/master/articles/_includes/_logout_url.md) and [allowed web origins](https://github.com/auth0/docs/blob/master/articles/_includes/_web_origins.md) snippets in 
 [getting started](https://github.com/auth0/docs/blob/master/articles/quickstart/spa/_includes/_getting_started.md) snippet, as is done in [VueJS QS](https://auth0.com/docs/quickstart/spa/vuejs#configure-logout-urls).

Reply:
I wasn't aware of this dynamic values.. Thanks for clarifying that 👍